aws_lambda_powertools/utilities/data_classes/code_pipeline_job_event.py

```diff
@@ -236,7 +236,25 @@ def find_input_artifact(self, artifact_name: str) -> CodePipelineArtifact | None
                 return artifact
         return None
 
-    def get_artifact(self, artifact_name: str, filename: str) -> str | None:
+    def find_output_artifact(self, artifact_name: str) -> CodePipelineArtifact | None:
+        """Find an output artifact by artifact name
```

Comment:
I'm flexible on this as this is the same behaviour as `find_input_artifact`.

Ultimately, i think that this should fail if the artifact isn't found, but I opted for consistency.

However: uploading an artifact _will_ fail if the artifact is not found as that seems necessary.